terraform/azure/postgres_provision.tf

```diff
@@ -0,0 +1,96 @@
+resource "azurerm_postgresql_server" "postgres" {
+  name                = "postgresql-server-obsrv"
+  location            = data.azurerm_resource_group.rg.location
+  resource_group_name = data.azurerm_resource_group.rg.name
+
+  sku_name = "GP_Gen5_2"
+
+  storage_mb                   = var.RDS_STORAGE
+  backup_retention_days        = 7
+  geo_redundant_backup_enabled = false
+  auto_grow_enabled            = true
+
+  administrator_login          = "druid"
```

Comment:
Let's variables all these details and please read the credentials either from the config or env variable. 
and don't keep any credentials details in the code.